.github/workflows/build_with_conda.yml

```
@@ -1,6 +1,13 @@
 name: Build with conda
 on:
+  # For Pull-Requests, this runs the CI on merge commit
+  # of HEAD with the target branch instead on HEAD, allowing
+  # testing against potential new states which might have
+  # been introduced in the target branch last commits.
+  # See: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
   push:
+  - master
```

As pointed out by @qc00 in https://github.com/man-group/ArcticDB/pull/390#discussion_r1195116324:
```suggestion
  push:
    branches:
      - 'master'
```